README.md

```diff
@@ -1,9 +1,9 @@
 # Java Core June 2021
 
-## *Nikolaev Artem*
+## *Prokofev Ilia*
 
-| Number | Solution  | Short description
+| Index | Solution  | Short description
 | --- | --- | --- |
-| HW1 | [Console printer](https://github.com/NikolaevArtem/Java_Core_June_2021/tree/master/src/main/java/homework_1) | The app that reads input arguments and prints them, until "error" argument |
+| HW1 | [Console printer](https://github.com/NikolaevArtem/Java_Core_June_2021/tree/feature/IliaProkofev/src/main/java/homework_1) | The app that reads input arguments and prints them, until "error" argument |
```

Comment:
It still looks at my branch

Reply:
I've no changed there anything, and it's looks at my branch _"tree/feature/IliaProkofev/src/main/java/homework_1"_ instead of of _"tree/master/src/main/java/homework_1"_

Reply:
You're right, sorry, I looked at wrong place